tools/deps/PackageLockWhitelistTest.java

```diff
@@ -42,6 +42,7 @@
   private List<String> packageLockLines;
   private List<Dependency> parsedDependencies;
 
+  @SuppressWarnings({"checkstyle:AbbreviationAsWordInName", "checkstyle:MemberName"})
   private String MAVEN_CENTRAL_URL = "https://repo.maven.apache.org/maven2/";
```

Comment:
This could be static, right?

Reply:
@oferb yup

Reply:
@oferb fixed in https://github.com/google/startup-os/pull/490/commits/8d25c3dbd20f74639a80a9f09be905132776aa1a